BrotliTesting/BrotliTesting.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you move this project into the "tests" directory?

If this is just for your use, please remove it from the pull request.